js/index.js

```diff
@@ -22,9 +24,10 @@ define([
     Adapt.once('xapi:lrs:initialize:error ' + successEvent, function() {
       Adapt.offlineStorage.get();
       Adapt.offlineStorage.setReadyStatus();
-      Adapt.wait.end();
     });
 
+    Adapt.wait.end();
```

Comment:
I'm not sure you need these waits at all now as the code is synchronous.

Reply:
I was just thinking this last night. I'll test out those changes and submit it if they are all good 👍

Reply:
Unfortunately offlineStorage:prepare isn't firing so I have had to stick with the app:dataLoaded as before